### DISCUSSION TITLE
Displaymode equivalent, scaling and math syntax

### DISCUSSION THREAD

**Discussion by Doublonmousse** [typst/typst#1441] [General]
Is there a typst equivalent to the latex ```\displaystyle``` command ?

I am used to doing ``` [text] \( \displaystyle [latex code] \) [more text]``` to have full size math writing without having to create a new line for the larger ```\[ [latex code\]``` and break the line.

For example 
![image](https://github.com/typst/typst/assets/115779707/2081e9b6-52cf-4309-90f9-03c3d5736677)

This is also useful in other situation, when writing math expressions in an itemized list.

Also I am very used to the latex \( \) and \[ \] commands to separate clearly the two syntax, and to create blocks of the form
```
\(
[tex code]
\)
```

and 
```
\[
[tex code]
\]
```
as I find it easier to visualize where math equation starts and end, and easier to format the text code to be more readable. So I would like typst to support these commands for math blocks. They can also be easier to parse compared to $ $ as the start and end of each bloc are easier to find.

**Comment by PgBiel** [comment]
Regarding display mode: This will come as `display(...)` in the next release (v0.5.0).
Regarding alternative syntax: This seems like it would conflict with the existing syntax for content (`[...]`) and for lists (`(...)`) in code mode. I think the way it currently is is fine (you can write `$  ...  $` in separate lines for example).
Alternatively, if you prefer, you can use `#[$ ... lines ... $]` to group things up in a way closer to what you propose.

**Reply by Doublonmousse** [reply]
Looking forward to the next release then!
